projects/canopy/src/lib/card/docs/guide.stories.mdx

```diff
@@ -80,6 +80,12 @@ It's possible to nest the Canopy layout grid within a card, as follows:
 
 <Source id="components-card-examples--nested-grid-card"></Source>
 
+### Card Navigation
+
+This component uses some extra card components, such as `LgCardNavigationTitleComponent` to display in navigational format.
+
+<Source id="components-card-examples--navigation-card"></Source>
+
 ## Additional development details
 
 ### LgCardComponent
```

Comment:
Could you please add the new component to the list below and add its `Input`s information?